qtpy/tests/test_qtcharts.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import PYSIDE2
+
+
+@pytest.mark.skipif(not PYSIDE2, reason="Only available in Qt5 bindings")
```

Comment:
`reason` needs to be changed here to

    reason="Only available by default in PySide2"